pkg/client/kubernetes/types.go

```diff
@@ -107,6 +111,9 @@ func init() {
 		druidv1alpha1.AddToScheme,
 		apiextensionsscheme.AddToScheme,
 		istionetworkingv1beta1.AddToScheme,
+		auditv1.AddToScheme,
+		auditv1beta1.AddToScheme,
+		apiserverv1alpha1.AddToScheme,
```

Comment:
Why do you add them here? Probably they only should be added to a dedicated scheme in the KAPI package?